roles/generate_ssh_key_pair/tasks/main.yml

```
@@ -19,6 +19,11 @@
     - "{{ key_pair_dir }}/{{ private_key_name }}"
     - "{{ key_pair_dir }}/{{ public_key_name }}"
 
+- name: Remove ssh_keys folder from /tmp bastion
```

Due to ansible's policy of everything can be overwritten could you make the name a template so its something like 
```suggestion
- name: "Remove temporary ssh_keys folder {{ key_pair_dir }}"
```

---

Yes, You're right